### ISSUE TITLE
why r sqaure is negative in compare model demo 

### ISSUE THREAD

**Issue by Yufei-Z** [pycaret/pycaret#552]
what is the definition of the r2 in compare_model ?
is it sign(corrcoef)* r2 in cv validation part?

**Comment by pycaret** [comment]
@Yufei-Z It uses `sklearn.metrics.r2_score`.

R-square can be negative. Read more: http://www.fairlynerdy.com/what-is-r-squared/

**Comment by Yufei-Z** [comment]
is it in train set or validation set?
